### PR TITLE
Clean title rewrites on DIP docs imports

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
## fixes KILTProtocol/ticket#3168

rewrites the imported DIP file titles to something more useful in the docs context.

### CODE REVIEW

**Comment by ntn-x2** [comment]
This is not the right changeset, given the PR description...

**Comment by ChrisChinchilla** [comment]
@ntn-x2 A rogue extra file snuck in, now good

File: docusaurus.config.js [hunk 1]
```diff
@@ -231,22 +231,21 @@ module.exports = {
...
       'docusaurus-plugin-remote-content',
       {
-        // options here
-        name: 'dip-provider-docs', // used by CLI, must be path safe
+        name: 'dip-provider-docs',
         sourceBaseUrl:
-          'https://raw.githubusercontent.com/KILTprotocol/kilt-node/release-1.12.1/pallets/pallet-dip-provider/', // the base url for the markdown (gets prepended to all of the documents when fetching)
-        outDir: 'docs/concepts/07_dip', // the base directory to output to.
-        documents: ['README.md'], // the file names to download,
+          'https://raw.githubusercontent.com/KILTprotocol/kilt-node/release-1.12.1/pallets/pallet-dip-provider/',
```
Comment (ntn-x2) [inline docusaurus.config.js]: We have release `1.13.0` now, so this should point to the new branch.

**Review by ntn-x2 — APPROVED** [review]
Nice!

**Pull request merged**